app/views/content/how-to-apply-for-teacher-training/teacher-training-personal-statement.md

```diff
@@ -41,73 +41,89 @@ quote:
+You can use the same personal statement for every course you apply for. There may be some instances where you’d like to tailor it to different courses.
 
-## Should my personal statement be different if I’m training to teach primary or secondary? 
+You can use your personal statement to explain why you feel passionate about teaching a specific age range or subject. You might also want to include why you want to do your teaching training with a particular provider.
 
-You should use your personal statement to explain why you feel passionate about teaching a specific age range or subject. 
+If you’re applying for a primary course with a subject specialism, or you’re particularly interested in certain primary subjects, you can include that in your statement.
 
-If you’re applying for a primary course with a subject specialism, or you’re particularly interested in certain primary subjects, you can talk about that, too. 
+If you’re not sure if you want to teach primary or secondary, you can [find out more about teaching different age groups](/life-as-a-teacher/age-groups-and-specialisms/age-groups-you-could-teach).
 
-If you’re not sure if you want to teach primary or secondary, you can [find out more about teaching different age groups](/life-as-a-teacher/age-groups-and-specialisms/age-groups-you-could-teach). 
+## Get help with your personal statement
 
-## Do I need school experience? 
+Writing a personal statement can feel daunting, but our teacher training advisers can help. 
 
-You do not need school experience to apply for teacher training, but it can help strengthen your personal statement. 
+Advisers can support you with:
 
-Teacher training providers like to see that you have a good understanding of teaching, how the school system works and what your transferable skills are. You need more than just good subject knowledge and school experience can be a great way to get this. 
+* what to include in your statement
 
-Getting some school experience can also be a good way to make sure teaching is right for you before you apply for a course. 
+* how to structure your statement so you can write it in managable sections
 
-[Find out how you could get school experience](/train-to-be-a-teacher/get-school-experience). 
+* developing your own voice in your statement  
 
-## Get help with your personal statement 
+$q-Josie$ 
 
-You can get help with your personal statement from our [teacher training advisers](/teacher-training-advisers). They have years of teaching experience and can give you free, one-to-one support by phone, text, or email. 
+You can get help with your personal statement from our [teacher training advisers](/teacher-training-advisers). They have years of teaching experience and can give you free, one-to-one support by phone, text, or email.
```

Comment:
Should this link be in the first sentence instead?

Or should there be a standalone link be at the end of the adviser section (with a green arrow!!)